lib/engine/action/base.rb

```diff
@@ -40,6 +41,7 @@ def to_h
           'entity_type' => type_s(entity),
           'id' => @id,
           'user' => @user,
+          'created_at' => @created_at,
```

Comment:
you gotta make sure to not save this to postgres

Reply:
Verified that `{}.merge` overwrites any duplicate keys with the new value.